test/jdk/java/lang/Class/ArrayType.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
2022?

Reply:
Copyright updated to cover 2022 (filed created in 2021); thanks,.